src/main/resources/locales/cy/pensions-list-chd-daily-cause-list.json

```diff
@@ -0,0 +1,22 @@
+{
+    "title": "Rhestr Achosion Dyddiol Pensiynau (Adran Siawnsri) ",
```

Comment:
Extra spaces at the end of title + heading

Reply:
Updated